test/jit/test_recursive_script.py

```diff
@@ -157,28 +158,32 @@ def a(x):
             # Make sure that no entries are left over from the previous failure
             FileCheck().check_count("is being compiled", 2).run(str(e))
 
-    @unittest.skipIf(True, "Class annotations are a thing in > 3.5, need to fix for < 3.7")
+    @unittest.skipIf(sys.version_info[:2] < (3, 7), "Class annotations are a thing in > 3.5, need to fix for < 3.7")
     def test_constants_with_final(self):
-        class M(torch.nn.Module):
-            # TODO: Use this (see below)
-            # x : torch.jit.Final[int]
+        class M1(torch.nn.Module):
+            x : torch.jit.Final[int]
 
             def __init__(self):
-                super(M, self).__init__()
+                super().__init__()
                 self.x = 2
 
             def forward(self, t):
                 return t + self.x
 
+        self.checkModule(M1(), (torch.randn(2, 2),))
 
-        # TODO: Fix this test so that we can actually define the class like
-        #   class M(torch.nn.Module):
-        #       x : torch.jit.Final[int]
-        M.__annotations__ = {'x': torch.jit.Final[int]}
+        if sys.version_info[:2] >= (3, 8):
+            class M2(torch.nn.Module):
+                x : typing.Final[int]
```

Comment:
The goal of this test is to use `torch.jit.Final`
```suggestion
          class M2(torch.nn.Module):
              x : torch.jin.Final[int]
```